Ruby/HighScores: Update of latest implementation (#1219)
exercism/website-copy: tracks/ruby/exercises/high-scores/mentoring.md

### Talking points
- `@scores` vs `scores`: This is the most important issue in this exercise. Beginners may _write_ the getter method/`attr_reader`, but _call_ the instance variable `@scores` still. This link explains some of the advantages of using the getter method internally in the class instead of directly referencing the instance variable: [LaunchSchool: accessor_methods](https://launchschool.com/books/oo_ruby/read/classes_and_objects_part1#accessormethodsinaction). It's recommended to link to this or your favorite explanation, instead of typing the solution for them. 

> - `latest_is_personal_best?`: this predicate method should use the existing methods, and implicitly return true or false. For new programmers it may be helpful to ask what they think the question mark is for.   

Reviewer: ```suggestion
- `latest_is_personal_best?`: this predicate method should use the existing methods, and implicitly return true or false. For new programmers it may be helpful to ask "What does the question mark indicate?"
```

Author: ```suggestion
- `latest_is_personal_best?`: this predicate method should use the existing methods, and implicitly return true or false. For new programmers it may be helpful to ask what they think the question mark indicates.   
```

Author: It's a compromise. Okay?

Reviewer: I guess it is too late to commit the suggestion in answer. lol  So I will take that as a rhetorical question.
